packages/pages-components/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "moduleResolution": "NodeNext",
     "outDir": "./dist",
     "target": "ESNext",
-    "types": ["vite/client", "node", "vitest/globals"],
+    "types": ["vite/client", "node", "vitest/globals", "bingmaps", "google.maps", "leaflet", "mapbox-gl"],
```

Comment:
Idk that these types need to be defined here

Reply:
I get a lot of ts errors in the `providers/` files if I don't include these

Reply:
Hmm, ok